util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialsConfiguration.java

```diff
@@ -341,7 +363,10 @@ private static GoogleCredentials configureCredentials(
     }
     if (credentials instanceof UserCredentials) {
       return ((UserCredentials) credentials)
-          .toBuilder().setTokenServerUri(URI.create(tokenServerUrl)).build();
+          .toBuilder()
+              .setAccessToken(credentials.getAccessToken())
```

Comment:
I had to do that, to workaround https://github.com/googleapis/google-auth-library-java/pull/993